Dqi improvements (#531)
akeneo/pim-helpcenter: content/md/products-super-power/improve-data-quality.md

From this panel, you will know which attributes need to be improved and how to improve them.

::: info

> From the 'data quality panel', links on attribute names can take you to a specific attribute. By clicking on the 'Display all improvable attributes', you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.

Reviewer: ```suggestion
From the `data quality panel`, links on attribute names can take you to a specific attribute. By clicking on the `Display all improvable attributes`, you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
```

> From the 'data quality panel', links on attribute names can take you to a specific attribute. By clicking on the 'Display all improvable attributes', you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.

Reviewer 2: ```suggestion
From the data quality panel, links on attribute names can take you to a specific attribute. By clicking on 'Display all improvable attributes', you can easily access a product edit form filtered on all the attributes that need to be checked for consistency.
```

:::

To help you even further, in the `Attributes` panel, you can select spelling corrections and title formatting improvements.

![Image data quality spellcheck corrections](../img/data-quality-spelling.png)

::: info
Spell check is available for the English, French, German, Spanish, Italian, Dutch, Danish, Swedish, Norwegian and Portuguese (Brazilian) locales.

> Spell check is usable with localisable text and text area attributes.

Reviewer 2: ```suggestion
Spell check is usable with localizable text and text area attributes.
```

:::

::: info
Title formatting suggestion is available for English locales.
:::

::: info
Even if they don't have permission to edit, all PIM users will be able to read data quality insights on all attributes.
:::

# Improve data quality on variant products

> If you work with variant products, you will see slight differences between grades and recommendations for simple, product models and variant products.

Reviewer 2: I feel there is a word missing here:  for simple products?

Author: yes, there is. thanks

For **product models**:
![Image data quality product model panel](../img/data-quality-model-panel.png)

> - Data quality grades are calculated on the attributes that can be improved at product model level.

Reviewer 2: ```suggestion
- Data quality grades are calculated on the attributes that can be improved at the product model level.
```

> - Recommendations are given for attributes at product model level.

Reviewer 2: ```suggestion
- Recommendations are given for attributes at the product model level.
```

> For **sub-product model**:

Reviewer: ```suggestion
For **sub-product models**:
```

![Image data quality sub product model panel](../img/data-quality-sub-model-panel.png)

> - Data quality grades are calculated on the attributes that can be improved at product model + sub-product model levels

Reviewer 2: ```suggestion
- Data quality grades are calculated based on the attributes that can be improved at the product model and sub-product model levels
```

> - Recommendations are given for attributes at product model + sub-product model levels by clicking on an attribute name, you can go and improve an attribute on the product model.

Reviewer: ```suggestion
- Recommendations are given for attributes at product model + sub-product model levels by clicking on an attribute name, you can go and improve an attribute on the product model or sub-product model.
```

Reviewer 2: ```suggestion
- Recommendations are given for attributes at the product model and sub-product model levels. By clicking on an attribute name, you can go and improve an attribute in the product model or sub-product model.
```

For **variant products**:
![Image data quality variant panel](../img/data-quality-variant-panel.png)

> - Data quality grades are calculated on the attributes that can be improved at all levels hence reflecting the data quality of the product that will be published. Data quality grades are therefore displayed in the left side bar only for product variant.

Reviewer: Je mettrai plus en avant cette partie-là, peut-être avec un bloc info ?

> - Recommendations are given for all attributes by clicking on an attribute name, you can go and improve an attribute on the product model or sub-product model.

Reviewer: ```suggestion
- Recommendations are given for all attributes by clicking on an attribute name, you can go and improve an attribute on the product model or sub-product model or variant product.
```

Reviewer 2: ```suggestion
- Recommendations are given for all attributes by clicking on an attribute name, you can go and improve an attribute in the product model, the sub-product model or the variant product.
```
